examples/examplesVersionDependency.ttl

```diff
@@ -26,10 +36,27 @@
 	] ;
 	.
 
+<https://ontologies.exco.com/unversionedOntology>
+	a owl:Ontology ;
+	skos:scopeNote "This ontology does not declare a version."^^xsd:string ;
+	skos:prefLabel "Unversioned Ontology"^^xsd:string ;
+	.
+
+<https://ontologies.exco.com/unconventionalVersioning>
+	a owl:Ontology ;
+	owl:versionIRI <https://ontologies.exco.com/unconventionalVersioning-RC2> ;
```

Comment:
```suggestion
	owl:versionIRI <https://ontologies.exco.com/nonStandardVersioning-RC2> ;
```

Reply:
Shouldn't each test case test only one parameter? I.e., as Michael notes, this one should declare a non-standard version "RC2".

Reply:
The actual `ver:uniqueText` _has_ to be compliant, since that's part of our spec. We are trying to find here ontologies which have a `versionIRI` which does not comply with the `<baseIRI><semanticVersion>` pattern we promote. I will add a normal semantic version to this ontology, so that only one 'violation' is being tested.

Reply:
Actually, I went ahead and added a non-standard `ver:hasVersion` and a check for that as well.